packages/component-library/src/components/plmg-slider/test/plmg-slider.e2e.ts

```diff
@@ -3,33 +3,149 @@ import { AxePuppeteer } from '@axe-core/puppeteer';
+describe('sets the range and initial valuess', () => {
+  it('sets min and max values', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<plmg-slider name="Range Slider" range-values="0.1, 100.31"></plmg-slider>'
+    );
+    const element = await page.find('plmg-slider >>> input[type="range"]');
+    expect(element).toEqualAttribute('min', '0.1');
+    expect(element).toEqualAttribute('max', '100.31');
+  });
+  it('sets the value to min if not default-value is set', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<plmg-slider name="Range Slider" range-values="0, 10"></plmg-slider>'
+    );
+    const element = await page.find('plmg-slider');
+    expect(element).toEqualAttribute('value', '0');
+  });
+  it('sets the inital value if default value is within range', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<plmg-slider name="Range Slider" range-values="0, 10" default-value="5"></plmg-slider>'
+    );
+    const element = await page.find('plmg-slider');
+    expect(element).toEqualAttribute('value', '5');
+  });
+  it('sets the value to the min if default value prop is above the allowed range', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<plmg-slider name="Range Slider" range-values="0, 10" default-value="10.001"></plmg-slider>'
+    );
+    const element = await page.find('plmg-slider');
+    expect(element).toEqualAttribute('value', '0');
+  });
+  it('set the value to the min if the default value prop is below the allowed range', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<plmg-slider name="Range Slider" range-values="0.1, 9.99" default-value="-0.1"></plmg-slider>'
+    );
+    const element = await page.find('plmg-slider');
+    expect(element).toEqualAttribute('value', '0.1');
+  });
+});
 
-    await page.setContent('<main>' + htmlContent + '</main>');
+describe('displays marks', () => {
+  it('displays marks by default', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<plmg-slider name="Range Slider" range-values="0, 10"></plmg-slider>'
+    );
+    const element = await page.find('plmg-slider >>>  .marks');
+    expect(element).not.toBe(null);
+  });
+  it('displays does not display marks', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<plmg-slider name="Range Slider" marks="false" range-values="0, 10"></plmg-slider>'
+    );
+    const element = await page.find('plmg-slider >>>  .marks');
+    expect(element).toBe(null);
+  });
+});
 
-    const results = await new AxePuppeteer(page as unknown as Page)
-      .disableRules([
-        'document-title',
-        'html-has-lang',
-        'landmark-one-main',
-        'page-has-heading-one',
-      ])
-      .analyze();
+describe('re-renders when props change', () => {
```

Comment:
I suggest adding "plmg-slider" to your test cases to be explicit they relate to this component.
```suggestion
describe('plmg-slider re-renders when props change', () => {
```